src/sdkV2.ts

```diff
@@ -103,4 +103,25 @@ export default class CegaEvmSDKV2 {
       ...overrides,
     });
   }
+
+  async bulkSettleVaultsDcs(
+    vaultAddresses: EvmAddress[],
+  ): Promise<ethers.providers.TransactionResponse> {
+    const cegaEntry = this.loadCegaEntry();
+    return cegaEntry.bulkSettleDCSVaults(vaultAddresses);
+  }
+
+  async bulkStartTradesDcs(
+    vaultAddresses: EvmAddress[],
+  ): Promise<ethers.providers.TransactionResponse> {
+    const cegaEntry = this.loadCegaEntry();
+    return cegaEntry.bulkStartDCSTrades(vaultAddresses);
+  }
+
+  async submitDisputeDcs(
+    vaultAddresses: EvmAddress[],
+  ): Promise<ethers.providers.TransactionResponse> {
+    const cegaEntry = this.loadCegaEntry();
+    return cegaEntry.submitDCSDispute(vaultAddresses);
```

Comment:
see: `src/abiV2/IDCSEntry.json` > `submitDispute` for ABI for this function

I don't see a `submitDCSDispute` function name available, there's just a `submitDispute` function according to the ABI.

Checking with SC team, but for this PR, maybe just update it to the current ABI:

should be `submitDispute` which takes in a single `vaultAddress` not an array

Reply:
Updated the function name and params